Wrappers/Python/test/test_functions.py

```diff
@@ -926,38 +926,51 @@ def test_Lipschitz4(self):
 
 class TestTotalVariation(unittest.TestCase):
 
+    def setUp(self) -> None:
```

Comment:
I'm not sure these should all be in the same set up if  each test only uses one version. It makes it hard to see what's going on, and it's a waste of time initialising everything for every test.

Reply:
yes, although it's not too much stuff to initialise and very lightweight

Reply:
I like to add this in the setUp for some cases because avoids repeating code.